faasmtools/build.py

```diff
@@ -54,14 +53,29 @@
 ]
 WASM_CXXFLAGS_SHARED = WASM_CFLAGS_SHARED
 
+# ----------
 # LDFLAGS
+# ----------
+
+# Flags for static libraries
 WASM_LDFLAGS = [
     "-static",
     "-Xlinker --no-gc-sections",
     "-Xlinker --stack-first",
     "-Xlinker --no-check-features",
 ]
 
+# Flags for executables
+WASM_EXE_LDFLAGS = [
```

Comment:
Copied this from the `func/CMakeLists`. Having them here as well is a first step towards faasm/cpp#6. I'll work on the issue at some point in the near future (hopefully).